examples/docs_snippets/docs_snippets/integrations/airflow/tutorial_rewrite.py

```diff
@@ -0,0 +1,64 @@
+# start_imports
+import os
```

Comment:
@rexledesma  I've got a more pure rewrite here using the bash commands. Im interested if you think this exposes too many dagster concepts? specifically the `ins` and propagating schedule info from `tags` has me concerned.

In the `tutorial_rewrite_dagster.py` file i've got a little more streamlined python-only rewrite but we might want to make an even more simplified version thats focused on being functionally equivalent? idk we might want to create our own airflow dag to do the migrating from to make it a little simpler